src/Web/Theory.hs

```diff
@@ -989,83 +973,30 @@ htmlThyPath :: RenderUrl      -- ^ The function for rendering Urls.
             -> TheoryPath     -- ^ Path to render
             -> String         -- ^ the lemma's plaintext
             -> Html
-htmlThyPath renderUrl renderImgUrl info path lPlaintext =   go path
```

Comment:
This chunk of code and the follow-up in the diff are tooo difficult to review by hand. But from what I see this is simple stuff, it's just that it is difficult to create good diff. Could you tell me if you've tested this?

Reply:
Right, I see the diff is weird. I did not try to rebase this file, I started it from scratch so everything should be functionally as it was.